projects/storefrontapp/src/environments/environment.ts

```diff
@@ -15,7 +15,9 @@ import { Environment } from './models/environment.model';
 
 export const environment: Environment = {
   production: false,
-  occBaseUrl: buildProcess.env.CX_BASE_URL,
+  occBaseUrl:
+    buildProcess.env.CX_BASE_URL ??
+    'https://api.cpce-teamtiger1-d1-public.model-t.cc.commerce.ondemand.com',
```

Comment:
don't forget to revert it back before merge.

Reply:
Picked

Reply:
Done